scene/gui/tree.cpp

```diff
@@ -5062,25 +5062,24 @@ String Tree::get_tooltip(const Point2 &p_pos) const {
 
 		if (it) {
 			const TreeItem::Cell &c = it->cells[col];
-			int col_width = get_column_width(col);
+			int col_width = get_size().width - theme_cache.panel_style->get_minimum_size().width - (h_scroll->is_visible_in_tree() ? h_scroll->get_minimum_size().width : 0);
```

Comment:
You could change it into conditional code like the one above, for better clarity.